src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -1045,10 +1045,12 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         WINRT_PROPERTY(winrt::hstring, Name, L"");
         WINRT_PROPERTY(Model::DesktopBehavior, Desktop, Model::DesktopBehavior::ToCurrent);
         WINRT_PROPERTY(bool, ToggleVisibility, true);
+        WINRT_PROPERTY(uint32_t, DropdownDuration, 0);

```

Comment:
Yea so now we're getting into the subjective realm - IMO the normal min/maximize behavior looks better than the dropdown. The animation seems more gimmicky - good for literally "quake mode", but seems more annoying for the others.

Reply:
I'm gonna tap in @DHowett and @cinnamon-msft, since: 
* this is the most contentious part of this PR
* _someone_ wants this by next week
* whenever there's a discussion about a setting, [it's always you three](https://i.pinimg.com/originals/ce/56/f3/ce56f30b698f66f94a5f30754994420c.jpg)

I'd feel more comfortable making dropdown the default if the whole window could animate. But since it can't then this almost feels more experimental. That's why I'm thinking we should leave the default `0`. It'll still play the normal Windows "minimize" and "restore" animations that a window would usually get with `"dropdownDuration": 0`.

Reply:
I think since the animation is requested, we can put it in for now. Then we can test it/play with it and if we don't like it we can just set it back to 0 in a follow up before 1.9?

Reply:
> it's always you three

lol